e-commerce-frontend/src/app/_services/user.service.ts

```diff
@@ -26,17 +27,17 @@ export class UserService {
 
   //get data for users with the 'SELLER' role
   public getDataForSeller() {
-    return this.httpClient.get(this.PATH_OF_API + "/forSeller", { responseType: 'text' });
+    return this.httpClient.get(this.PATH_OF_API + "seller/forSeller", { responseType: 'text' });
```

Comment:
I saw that you have a different structure of http requests than `product.service.ts`. Try to stick to one kind of implementation. Also, the `responseType` should not be `json`?